.github/workflows/ci.yml

```diff
@@ -87,6 +88,10 @@ jobs:
           api-key: ${{ secrets.FORESIGHT_API_KEY }}
           surefire-reports-path: 'testsuite/integration-arquillian/tests/base/target/surefire-reports/*.xml'
 
+      - uses: ./.github/actions/upload-flaky-tests
```

Comment:
Looking at the output on GHA, it doesn't use the name in the included file. 

To have a nice name in the summary, please add a name here.

Reply:
Going to add `name: Upload flaky tests`